src/AppInstallerRepositoryCore/Rest/HttpClientHelper.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include "pch.h"
+#include "cpprest/http_client.h"
+#include "cpprest/json.h"
+#include "HttpClientHelper.h"
+
+namespace AppInstaller::Repository::Rest
+{
+	HttpClientHelper::HttpClientHelper(utility::string_t url) : m_client(url), m_url(url) {}

```

Comment:
>	 [](start = 0, length = 1)

Another reason to use CF, easier to see tabs.  Please convert to 4 spaces.

Reply:
I have modified my settings to use spaces, but still see some tabs. Have replaced them, let me know if there are more. Btw, Do you have a plugin for codeflow that detects tabs? 

---
In reply to: [586819753](https://github.com/microsoft/winget-cli/pull/765#discussion_r586819753) [](ancestors = 586819753)